### PR TITLE
ICON 2022 ingestion

### PR DESCRIPTION
**Pull request by xinru1414**

### CODE REVIEW

File: data/yaml/venues/nalm.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+acronym: nalm
```
Comment (mjpost) [inline data/yaml/venues/nalm.yaml:1]: should be NALM for the acronym

File: data/xml/2022.nalm.xml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    <paper id="2">
+      <title>A custom <fixed-case>CNN</fixed-case> model for detection of rice disease under complex environment</title>
+      <author><first>Chiranjit</first><last>Pal</last></author>
+      <author><first>Sanjoy</first><last>Pratihar</last></author>
+      <author><first>Imon</first><last>Mukherjee</last></author>
+      <pages>5-8</pages>
+      <abstract>The work in this paper designs an image-based rice disease detection framework that takes rice plant image as input and identifies the presence of BrownSpot disease in the image fed into the system. A CNN-based disease detection scheme performs the binary classification task on our custom dataset containing 2223 images of healthy and unhealthy classes under complex environments. Experimental results show that our system is able to achieve consistently satisfactory results in performing disease detection tasks. Furthermore, the CNN disease detection model compares with state-of-the-art works and procures an accuracy of 96.8%.</abstract>
+      <url hash="e2beece6">2022.nalm-1.2</url>
+      <bibkey>pal-etal-2022-custom</bibkey>
+    </paper>
```
Comment (mjpost) [inline data/xml/2022.nalm.xml:37]: please add `<event>` block and list workshops

**Review by mjpost — COMMENTED** [review]
Please also add the workshops to `2022.ws.xml` so they appear there, as well.

**Comment by as3eem** [comment]
Hi @xinru1414 @mjpost,
As per a recent request from the organizers, a paper seems to be repeating in the shared task volume, which makes the in-place paper missing. Can we correct this and give you the updated output. Also there is a small typo in the main volume. Will update that too. Let me know.

**Comment by xinru1414** [comment]
@as3eem Please share with us the updated files, we can reingest then.

**Comment by xinru1414** [comment]
@as3eem Any update on this?

**Comment by as3eem** [comment]
@xinru1414 the same drive folder has the updated files. 

https://drive.google.com/drive/folders/1DKZrrXuDWdo6hECgZd8EhXDejrPX0QrW?usp=sharing

**Comment by xinru1414** [comment]
@as3eem Can you confirm that only the main volume "Proceedings of the 19th International Conference on Natural Language Processing (ICON)" and the shared task "Proceeding of the Shared Task" have updates (i.e. need reingestion), and the two workshop proceedings are all set?

**Comment by as3eem** [comment]
@xinru1414  the main volume and the shared task.

**Comment by mjpost** [comment]
I'll review once @xinru1414 notes it's complete. Then we can merge.

**Comment by anthology-assist** [comment]
@mjpost Ready for review and merge.

**Comment by as3eem** [comment]
@anthology-assist @xinru1414 @mjpost 
I am continuously getting emails requesting updates on the proceeding. Can you provide an ETA on this?

**Comment by as3eem** [comment]
Requesting @mjpost to expedite the process. I am not sure what is taking so long. Kindly let me know if there is anything I can do to make the process easy for you.

Also requesting @anthology-assist and @xinru1414 to help in this matter kindly.

**Comment by mjpost** [comment]
[Github is down](https://www.githubstatus.com), will merge when everything is restored.

**Comment by mjpost** [comment]
[Github is down](https://www.githubstatus.com), will merge when everything is restored.

**Comment by as3eem** [comment]
Hi @mjpost @xinru1414,

Thanks for your support in making the proceeding live. However, the anthology shows only the main and the shared task's proceeding. We do not see any workshop proceeding. May I know if the process is still under process? 

I am asking as I see the PR has been merged.

**Comment by mjpost** [comment]
You are looking at the volumes page which shows only ICON volumes. You want the event page: https://aclanthology.org/events/icon-2022/

**Pull request merged**